src/data/un-sdsn-cw/SDSN-CW-OEP-LA-2025.json

```
@@ -0,0 +1,61 @@
+  "metric": ["Absolute Emissions", "Capacity", "Generation", "Technology Mix"],
+  "keyFeatures": {
+    "emissionsTrajectory": "Significant increase",
+    "energyEfficiency": "No information",
+    "energyDemand": "No information",
+    "electrification": "Moderate increase",
+    "policyTypes": ["Target technology shares"],
+    "technologyCostTrend": "Decrease",
+    "technologyDeploymentTrend": "No information",
+    "emissionsScope": "CO2e (Kyoto)",
+    "policyAmbition": "Normative/Optimization-based",
```

```suggestion
    "policyAmbition": "Current/legislated policies",
```
this is wrong, updated the source tab